e2e/utils/graphql.ts

```diff
@@ -33,6 +33,27 @@ const createGaloyServerClient = (config) => (authToken) => {
   })
 }
 
+const authTokens = process.env.GALOY_TEST_TOKENS?.split(",") || []
+
+const randomizeTokens = (arr: string[]): string => {
```

Comment:
this function is overly complicated.

here is chatGPT rewrite of the function:

```
const randomizeTokens = (arr: string[]): string => {
  const randomIndex = Math.floor(Math.random() * arr.length);
  return arr[randomIndex];
}
```